### DISCUSSION TITLE
Please add native support for CalDAV URL Calendars. And allow to remove the default event type.

### DISCUSSION THREAD

**Discussion by hlarc** [FossifyOrg/General-Discussion#426] [Ideas]
I setup Radicale to host my own Calendar and wanted to use the Fossify Calendar App but i cant natively sync from and to my server. I also cant create Events, because in the Selection Window for the event type my CalDAV Calendars dont show up. (But they are rw calendars) I used ICSx^5 for the CalDAV Synchronisation as a Workaround. And the other would be to allow the deletion (or to hide it) of the default Event Type. Because I only use Network Calendars for example.

**Comment by Aga-C** [comment]
> I setup Radicale to host my own Calendar and wanted to use the Fossify Calendar App but i cant natively sync from and to my server.

By design, Fossify apps have no Internet access, so for CalDAV synchronization we will always rely on external apps like DAVx5.

>  I also cant create Events, because in the Selection Window for the event type my CalDAV Calendars dont show up. (But they are rw calendars)

Are you creating events or tasks? Because we currently don't support tasks synchronization.

> And the other would be to allow the deletion (or to hide it) of the default Event Type. Because I only use Network Calendars for example.

It's not possible. See: https://github.com/FossifyOrg/Calendar/issues/150#issuecomment-1946167183.

**Reply by hlarc** [reply]
> By design, Fossify apps have no Internet access, so for CalDAV synchronization we will always rely on external apps like DAVx5.

Well thats unfortunate. But not an issue. Thanks for the Info.
 
> Are you creating events or tasks? Because we currently don't support tasks synchronization.

I created an Event and clicked on the Button with the geometrical shapes to change the calendar where its stored But none of my CalDAV Calendars showed up.

> It's not possible. See: [#150 (comment)](https://github.com/FossifyOrg/Calendar/issues/150#issuecomment-1946167183).

Ok if it cant be deleted. But can you still Hide it from the calendar view? Or maybe add a third option to the manage Calendars Menu, where you can hide them and they only show up in the in the manage calendars menu (not on the calendar page and not in the menu to select in wich calendar the event should be created in)

**Comment by Aga-C** [comment]
> I created an Event and clicked on the Button with the geometrical shapes to change the calendar where its stored But none of my CalDAV Calendars showed up.

If you have turned on CalDAV synchronization, just above the field you're describing, you should see a field with calendar icon with text `Store locally only`. Tap on it and choose your calendar.

> But can you still Hide it from the calendar view? 

You can hide it in Settings → Manage quick filter event types.

**Reply by hlarc** [reply]
> If you have turned on CalDAV synchronization, just above the field you're describing, you should see a field with calendar icon with text `Store locally only`. Tap on it and choose your calendar.

Well it opens a new menu with one option "store locally only" wich is ticked.
 
> You can hide it in Settings → Manage quick filter event types.
Thanks for the Info works for me.  :)

**Reply by Aga-C** [reply]
> Well it opens a new menu with one option "store locally only" wich is ticked.

Just to be sure: in Settings → Manage synced calendars, do you have checked any calendar?

**Reply by hlarc** [reply]
> Just to be sure: in Settings → Manage synced calendars, do you have checked any calendar?

Yes I checked the four calendars i want to sync and they even show up in the Quick Filter but i cant add to the calendar. And i checked the URLs to see if i can edit the calendar and in Thunderbird it worked.

**Reply by Aga-C** [reply]
Thanks! It seems to be a bug in the app. Could you create a [bug report](https://github.com/FossifyOrg/Calendar/issues/new?assignees=&labels=bug%2Cneeds+triage&projects=&template=bug_report.yml) adding as many details as you can?

**Reply by opqx135** [reply]
This can be solved by using the DAVx^5 connector (NOT the ICSx^5 connector) instead, this is also available on F-Droid.

**Reply by KatharineRosewood** [reply]
@opqx135 This worked for me i finally got my local radicale server on my pc to work with my calendar on my phone by using davx5 instead of icsx5
